src/prefabs/pageWithCrudSlideOutPanel.tsx

```diff
@@ -2853,6 +2854,34 @@ const beforeCreate = ({
     },
   });
 
+  const disabledKinds = createBlacklist([
```

Comment:
login token should probably not have to be whitelisted here, object property likely not supported yet as well